dash/development/_r_components_generation.py

```diff
@@ -821,3 +855,13 @@ def create_prop_docstring_r(prop_name, type_object, required, description,
                    ': {}'.format(description) if description != '' else ''
                ),
                is_required='required' if required else 'optional')
+
+
+def get_wildcards_r(prop_keys):
+    wildcards = ""
+    wildcards += ", ".join("'{}'".format(p)
+                           for p in prop_keys if "*" in p)
```

Comment:
again, for consistency `p.endswith("-*")`

Reply:
fixed in 1578e9ac76316be72f211e2a318ee764707243b4